### PR TITLE
move gazebo tags to a separate file. clear up URDF

### PR DESCRIPTION
**Pull request by shokoofeh**

### CODE REVIEW

File: grizzly_description/urdf/base.urdf.xacro [hunk 1]
```diff
@@ -28,26 +28,14 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
-       xmlns:body="http://playerstage.sourceforge.net/gazebo/xmlschema/#body"
-       xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom"
-       xmlns:joint="http://playerstage.sourceforge.net/gazebo/xmlschema/#joint"
-       xmlns:controller="http://playerstage.sourceforge.net/gazebo/xmlschema/#controller"
-       xmlns:interface="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface"
-       xmlns:rendering="http://playerstage.sourceforge.net/gazebo/xmlschema/#rendering"
-       xmlns:renderable="http://playerstage.sourceforge.net/gazebo/xmlschema/#renderable"
-       xmlns:physics="http://playerstage.sourceforge.net/gazebo/xmlschema/#physics"
-       xmlns:xacro="http://ros.org/wiki/xacro"
-       name="grizzly">
```
Comment (mikepurvis) [inline grizzly_description/urdf/base.urdf.xacro:44]: Good riddance, what a mess.

**Comment by mikepurvis** [comment]
This all looks good to me, but I don't really know how to reason about the `turnGravityOff` and `selfCollide` stuff. @TheDash, can you shed some light?

**Comment by UltronDestroyer** [comment]
Can't remember about the gravity thing, the self collide thing makes sure
the base link's successor links aren't in collision with eachother.

On Wed, Dec 10, 2014 at 2:27 PM, Mike Purvis notifications@github.com
wrote:

> This all looks good to me, but I don't really know how to reason about the
> turnGravityOff and selfCollide stuff. @TheDash
> https://github.com/TheDash, can you shed some light?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/g/grizzly/pull/7#issuecomment-66508331.

## 

Devon Ash
<a href="http://ca.linkedin.com/pub/devon-ash/48/478/981"
style="text-decoration:none;"><span style="font: 80% Arial,sans-serif;
color:#0783B6;">View Devon Ash's LinkedIn profileView Devon Ash's profile</span></a>

**Comment by mikepurvis** [comment]
Okay, merge this for now; those things can be adjusted as experiment demands.

**Pull request merged**